### DISCUSSION TITLE
Seeking Guidance for Translating Postal: Best Practices?

### DISCUSSION THREAD

**Discussion by leoborlot** [postalserver/postal#2504] [Help with using Postal]
Hello everyone,

I hope this message finds you well.

I am currently working on a project involving the Postal and I've encountered a roadblock. I'm trying to understand how to execute a translation of Postal for Portuguese BR, but I'm a bit unsure about the best way to approach it.

If anyone has had any experience with this before or could offer any advice, it would be greatly appreciated. I am especially interested in best practices for carrying out this task to ensure the most effective and efficient process. How do you usually handle the translation? Are there any tools or techniques that you'd recommend for this specific task?

I understand that it might take time to respond to this, so I really appreciate any help or guidance you can provide.

Thank you so much in advance for your time and assistance.

Best regards,

**Comment by willpower232** [comment]
Unfortunately Postal does not seem to be set up for internationalisation at the moment, as per [the docs](https://guides.rubyonrails.org/i18n.html) it looks like calls to `I18n.t` are needed everywhere and then the translation YAML files could be written.
